src/traverse/__tests__/node-path-test.js

```diff
@@ -368,7 +429,7 @@ describe('NodePath', () => {
     expect(dPath.parentPath).toBe(alterPath);
     expect(alterPath.parentPath).toBe(groupPath);
 
-    expect(generator.generate(ast)).toBe('/a(cd)/');
+    expect(generator.generate(ast)).toEqual('/a(cd)/');
```

Comment:
nit for the future: `toBe` is usually fine with primitive values (like strings)

Usually, `toBe` is used to check the "identity", and `toEqual` to check a complex "shape" of an object.

```js
const foo = {x: 1};

// ok, same memory location
expect(foo).toBe(foo);

// fails, since two different objects in memory
expect(foo).toBe({x: 1});

 // ok, shapes are the same
expect(foo).toEqual({x: 1});
```

Since strings are primitive data type, they are always equal with `toBe` (and technically with `toEqual`).